pyramid_zipkin/request_helper.py

```diff
@@ -160,6 +159,8 @@ def get_binary_annotations(request, response):
         'http.uri.qs': request.path_qs,
         'response_status_code': str(response.status_code),
     }
+    if 'version' in request.registry:
+        annotations['version_SHA'] = request.registry.get('version')
```

Comment:
i prefer to just access it directly (like request.registry.get['version']) since you've already checked for its existence